src/pages/installing-spark/html-environment-setup.mdx

```diff
@@ -127,18 +142,16 @@ with Webpack:
 },
 ```
 
-Webpack will combine all JavaScript dependencies into a single
-JavaScript file called `main.js`. This is the file that you want
-your site to use.
+Webpack will combine all JavaScript into a single
+file called `main.js`, so we'll need to reference that instead of `index.js`.
```

Comment:
This needs to be `"build": "webpack --mode='development'"`. Github won't let me make it a suggestion.

Reply:
done!

![image](https://user-images.githubusercontent.com/4342363/120391373-eb4a6300-c2fc-11eb-9aa1-07c8b3070e5e.png)